Libs/MRML/Core/Testing/vtkMRMLStreamingVolumeNodeTest1.cxx

```diff
@@ -0,0 +1,117 @@
+/*==============================================================================
+
+Copyright (c) Laboratory for Percutaneous Surgery (PerkLab)
+Queen's University, Kingston, ON, Canada. All Rights Reserved.
+
+See COPYRIGHT.txt
+or http://www.slicer.org/copyright/copyright.txt for details.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+This file was originally developed by Kyle Sunderland, PerkLab, Queen's University
+and was supported through CANARIE's Research Software Program, and Cancer
+Care Ontario.
+
+==============================================================================*/
+
+#include "vtkMRMLCoreTestingMacros.h"
+#include "vtkMRMLStreamingVolumeNode.h"
+
+// vtkAddons includes
+#include "vtkStreamingVolumeCodecFactory.h"
+
+
+int vtkMRMLStreamingVolumeNodeTest1(int , char * [] )
+{
+  vtkNew<vtkMRMLStreamingVolumeNode> node1;
+  EXERCISE_ALL_BASIC_MRML_METHODS(node1.GetPointer());
+
+
+  vtkSmartPointer<vtkStreamingVolumeCodecFactory> factory = vtkStreamingVolumeCodecFactory::GetInstance();
```

Comment:
We don't use factory explicitly. Is it necessary to instantiate it then? We can keep it but then use it (e.g., check if there are codecs registered, etc).